index.js

```diff
@@ -153,6 +156,10 @@ Liftoff.prototype.buildEnvironment = function (opts) {
     modulePackage: modulePackage || {},
     configFiles: configFiles
   };
+
+  opts.argv = defaultValue(normalize.object(opts.argv, env), process.argv);
```

Comment:
Interesting but we should do this without normalize.  Just check if it's a value or a function or use the default.

Reply:
I got it.